pkg/storageos/rbac.go

```diff
@@ -6,6 +6,28 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// Exported role, binding and service account resource names.
+const (
+	DaemonsetSA   = "storageos-daemonset-sa"
+	StatefulsetSA = "storageos-statefulset-sa"
+
+	CSIProvisionerClusterRoleName    = "storageos:csi-provisioner-role"
+	CSIProvisionerClusterBindingName = "storageos:csi-provisioner-binding"
```

Comment:
I think we can remove the `-role` and `-binding` suffixes?

Reply:
Right, we can. I'll change it.